contract_sale/security/account_analytic_account_security.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account_analytic_account_own_salesman" model="ir.rule">
+        <field name="name">See Own Contracts</field>
+        <field name="model_id" ref="analytic.model_account_analytic_account"/>
+        <field name="domain_force">
+            ['|', ('user_id','=',user.id),
+            ('user_id','=',False),]</field>
+        <field name="groups"
+               eval="[(4,ref('sales_team.group_sale_salesman'))]"/>
+    </record>
+    <record id="account_analytic_account_see_all" model="ir.rule">
+        <field name="name">See All Contracts</field>
+        <field name="model_id" ref="analytic.model_account_analytic_account"/>
+        <field name="domain_force">[(1,'=',1)]</field>
+        <field name="groups"
+               eval="[(4,ref('sales_team.group_sale_salesman_all_leads'))]"/>
```

Comment:
Add here also the accounting roles, or they can be cut the permissions if they belong to `group_sale_salesman`